ddtrace/filters.py

```diff
@@ -9,7 +9,7 @@
 from .ext import http
 
 
-class TraceFilter(six.with_metaclass(abc.ABCMeta)):
+class TraceFilter(six.with_metaclass(abc.ABCMeta)):  # type: ignore[misc]
```

Comment:
What was the type error here?

Reply:
Seems like all the ABCs have type errors 🤔

Reply:
Here's the mypy error: `error: Unsupported dynamic base class "six.with_metaclass"  [misc]` This might be an issue with the ABCMeta rather than `with_metaclass`